internal/auth_test.go

```diff
@@ -0,0 +1,149 @@
+package internal
+
+import (
+	"strconv"
+	"testing"
+
+	"github.com/databricks/databricks-sdk-go"
+	"github.com/databricks/databricks-sdk-go/apierr"
+	"github.com/databricks/databricks-sdk-go/service/iam"
+	"github.com/databricks/databricks-sdk-go/service/oauth2"
+	"github.com/stretchr/testify/require"
+)
+
+func TestUcAccWifAuth(t *testing.T) {
```

Comment:
We should probably add a comment that these tests cannot run locally, or at least skip if it is run locally? Maybe check for the GITHUB_ACTIONS environment variable and skip if it is not set?